contrib/completion/zsh/_docker

```diff
@@ -217,6 +217,64 @@ __docker_search() {
     _wanted dockersearch expl 'available images' compadd -a result
 }
 
+__docker_run_all_capabilities() {
+    # list and description taken from https://docs.docker.com/engine/reference/run/#runtime-privilege-and-linux-capabilities
+    local -a default_off_capabilities default_on_capabilities all_capabilities
+    default_off_capabilities=(
+        "SYS_MODULE:Load and unload kernel modules."
```

Comment:
Could you

- sort these lists alphabetically, 
- also add `ALL` to the list? 
- double check if this list matches the Bash completion; https://github.com/docker/cli/blob/master/contrib/completion/bash/docker#L834-L882
-  Perhaps change the names of `default_off_capabilities` and `default_on_capabilities` to `capabilities_droppable` / `capabilities_droppable` / `capabilities_addable` (to match the bash completion)?

Reply:
> sort alphabetically
sure

> add `ALL` to the list
not sure i understand the suggestion. in [line 264 and 265](https://github.com/docker/cli/pull/2485/files#diff-b100a71a8a5e8e4eb980e60204bb5f6eR264) `ALL` gets added to the suggestions in any case, so I handle this special capability separate instead of adding it to both lists. Did you overlook that or do you suggest dropping the special treatment and adding `ALL` to the default_off and default_on lists?

> double check against bash
will do

> change the names
I like your suggestion

Reply:
✓ sorted alphabetically
✓ compared against bash: the only difference I spotted is that my alphabetic sorting put SYSLOG into a different position
✓ renamed the arrays (and rephrased the labels)
→ waiting for clarification on `ALL`. I thought back why I treated `ALL` separately in the first draft. I had decided in favor of that when I was considering to obtain the list of capabilities in an automated way rather than hardcoding them. Since I didn't go for that, the reason for three instead of two arrays is gone. Having a "suggest for drop" "suggest for add" and "suggest always" list kind of eases things up when completing `docker run --cap-add=ALL --cap-drop=` (because then `ALL` doesn't appear twice in the suggestions (in the "droppable (on by default)" and the "droppable (enabled through ALL)") but if you prefer two lists for other reasons that's fine with me